packages/zend-file-transfer/library/Zend/File/Transfer.php

```diff
@@ -22,7 +22,6 @@
 /**
  * @see Zend_Loader
  */
-// require_once 'Zend/Loader.php';
```

Comment:
do not remove commented out `require_once`s, please, as they are simply left there to be in all other occurrences - I would not do any code administrative/beautifying/cleaning tasks in the whole framework and keep the code changes only to the really necessary ones.